tests/data/ar/Item.php

```diff
@@ -20,4 +20,11 @@ public function getCategory()
     {
         return $this->hasOne(Category::className(), ['id' => 'category_id']);
     }
+
+    //relation with dynamic alias
+    public function getCategoryWithAlias()
```

Comment:
I am testing a relation with a dynamic alias (that doesn't exist), not a "regular" relation.